versions.json

```diff
@@ -236,12 +219,12 @@
     },
     "rdma_core": {
         "common": {
-            "branch": "stable-v48"
+            "branch": "v52.0"
```

Comment:
Just going with the latest version, not necessarily stable (as compared to before)?

Reply:
using tags rather than branches (maybe should rename this properties) for cleaner versioning.

That said, I see they have a `-2` release and so I see the argument of putting it back in.

Reply:
leaving as "branch", as we have the flexibility to use either here and I think we've needed to